src/screens/view-book/view-book.component.js

```diff
@@ -7,33 +7,67 @@ import {
   Content,
   Text,
   Button,
-  View
+  View,
+  Icon,
+  ActionSheet,
+  Spinner
 } from 'native-base';
 
 import {
-  Image
+  Image,
+  Alert
 } from 'react-native';
 
 import styles from './view-book.styles';
 
 import GoBack from '../../components/go-back/go-back.component';
 import TaskList from '../../components/task-list/task-list.component';
 
+var BUTTONS = ['Sim', 'Não']
+var YES_INDEX = 1;
+var NO_INDEX = 2;
```

Comment:
Se for constante prefira usar const e no lugar de var use let por ser block scoped coisa que var não é.
E eu acho que essas variáveis não estão sendo usadas certo ?